kedro/templates/project/hooks/utils.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+import shutil
+import sys
+import click
+
+current_dir = Path.cwd()
+
+lint_requirements = "black~=22.12.0\nruff~=0.0.290\n"
+lint_pyproject_requirements = """
+[tool.ruff]
+select = [
+    "F",  # Pyflakes
+    "E",  # Pycodestyle
+    "W",  # Pycodestyle
+    "UP",  # pyupgrade
+    "I",  # isort
+    "PL", # Pylint
+]
+ignore = ["E501"]  # Black takes care of line-too-long
+"""
+
+test_requirements = "pytest-cov~=3.0\npytest-mock>=1.7.1, <2.0\npytest~=7.2"
+test_pyproject_requirements = """
+[tool.pytest.ini_options]
+addopts = \"\"\"
+--cov-report term-missing \\
+--cov src/{{ cookiecutter.python_package }} -ra
+\"\"\"
+
+[tool.coverage.report]
+fail_under = 0
+show_missing = true
+exclude_lines = ["pragma: no cover", "raise NotImplementedError"]
+"""
+
+docs_pyproject_requirements = """
+docs = [
+    "docutils<0.18.0",
+    "sphinx~=3.4.3",
+    "sphinx_rtd_theme==0.5.1",
+    "nbsphinx==0.8.1",
+    "nbstripout~=0.4",
```

Comment:
This can now be removed because https://github.com/kedro-org/kedro/pull/3065 and https://github.com/kedro-org/kedro/pull/3072 have been merged.